ansible/roles/debops.dovecot/templates/etc/ansible/facts.d/dovecot.fact.j2

```diff
@@ -0,0 +1,26 @@
+#!{{ ansible_python['executable'] }}
+
+# {{ ansible_managed }}
+
+from __future__ import print_function
+from json import load, loads, dumps
+from sys import exit
+import subprocess
+import signal
+import os
+
+output = {'installed': True}
```

Comment:
This fact could be more dynamic by checking the existence of the `dovecot` command during execution. Check the fact of the `debops.ansible` role for a good example.